RELEASE.md

```diff
@@ -0,0 +1,49 @@
+## Release to Maven Central
```

Comment:
I think Travis should do this for certain triggers.
TODO for me to dig deeper into this.

Reply:
There was actually a recent article mentioned in the latest ScalaTimes about this: https://leobenkel.com/2019/09/travis-continuous-deployment/.So yes, you can do this automatically, personally I do not like a release for every PR and everything else (release branch etc.) would be kind of semi-automatic again. But that is of course up to you.

Reply:
Yepp I quickly read that article, when the email came. I think we will go with some semi-automatic method in the near future, but for now, I will merge this and try to push a version to the snapshot repo.